src/postamble_modularize.js

```diff
@@ -18,7 +18,7 @@ moduleRtn = Module;
 
 #endif // WASM_ASYNC_COMPILATION
 
-#if ASSERTIONS
+#if ASSERTIONS && MODULARIZE != 'static'
```

Comment:
Why not use `-sMODULARIZE=instance` to match the old `-sMODULARIZE_INSTANCE` setting?  Do you think the word "static" is more descriptive of what is happening here and "instance"?  I think of the two modes as "single instance" vs "factory/multiple instance", but maybe "static" makes sense on some level?

Reply:
I open to changing, but instance to me confusing. When I hear instance, it makes me think I can create multiple instances i.e. a factory. I chose static since the exports to the outside world don't change and there's only one static instance.

Other ideas: static_instance, singleton, esmodule. Or alternatively, we add a whole different flag that will enable all the settings we want in the "new output world" e.g. strict, es modules, no _ exports....

Reply:
I'm kind of leaning towards bring back the old name `-sMODULARIZE_INSTANCE` until we can think of a better one.    It is exactly the same meaning as the old option I believe, right?

Its interesting to me that the word `instance` to you implies multiple instances.  I don't think it has the connotation for me.

Reply:
I've renamed it instance for now. Later I'd like to look into the idea of having two independent options of `FACTORY(aka MODULARIZE)` and then `MODULE_FORMAT` = `none/esm/umd/...` as mentioned below and in the other bug.